docs/ops/shape/Squeeze_1.md

```diff
@@ -4,29 +4,33 @@
 
 **Category**: Shape manipulation
 
-**Short description**: *Squeeze* removes specified dimensions (second input) equal to 1 of the first input tensor. If the second input is omitted then all dimensions equal to 1 are removed. If the specified dimension is not equal to one then error is raised.
+**Short description**: *Squeeze* removes dimensions equal to 1 from the first input tensor.
+
+**Detailed description**: *Squeeze* can be used with or without the second input tensor.
+*       If only the first input is provided, every dimension that is equal to 1 will be removed from it.
+*       With the second input provided, each value is an index of a dimension from the first tensor that is to be removed. Specified dimension has to be equal to 1, otherwise an error will be raised. Dimension indices can be specified directly, or by negative indices (counting dimensions from the end).
 
 **Attributes**: *Squeeze* operation doesn't have attributes.
 
 **Inputs**:
 
-*   **1**: Multidimensional input tensor of type *T*. *Required*.
+*   **1**: Multidimensional input tensor of type *T*. **Required**.
 
-*   **2**: 0D or 1D tensor of type *T_SHAPE* with dimensions indices to squeeze. Values could be negative. *Optional*.
+*   **2**: 0D or 1D tensor of type *T_I* with indices of dimensions to squeeze. Values could be negative (have to be from range `[-R, R-1]`, where `R` is the rank of the first input). **Optional**.
 
 **Outputs**:
 
 *   **1**: Tensor with squeezed values of type *T*.
 
 **Types**
 
-* *T*: supported type.
+* *T*: any supported type.
 
-* *T_SHAPE*: supported integer type.
+* *T_I*: any supported integer type.
```

Comment:
I use `T_INT` for such tensor. Not sure which approach is better.

Reply:
I suppose both are better than `T_SHAPE`, I will use you suggestion.